apps/accounts/views.py

```diff
@@ -13,9 +13,11 @@
 from rest_framework import generics
 from rest_framework.response import Response
 from rest_framework.views import APIView
+from tower import ugettext as _
```

Comment:
This has to be `from tower import ugettext_lazy as _` otherwise you will be localizing everything only once -- when the app starts up and imports the module

![shower-cry-tobias](https://cloud.githubusercontent.com/assets/55398/12825674/1d1073ec-cb3d-11e5-837e-82b3bad68b9f.gif)